.gitignore

```diff
@@ -59,6 +59,7 @@ obj/
 # Project Specific List #
 # ===================== #
 artifacts/
+Assets/MRTK/**/.npmrc
```

Comment:
Does the script leave .npmrc files behind in the tree, or should it clean them up and this wouldn't be needed?

Reply:
it does not leave them behind. i can remove that entry

Reply:
removed.